### PR TITLE
Fix test deprecation warning on 0.6

### PR DESCRIPTION
**Pull request by TotalVerb**
Also bumps Compat dependency to <s>0.8.5</s> 0.9.5.

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,2 +1,2 @@
 julia 0.4
-Compat 0.8.4
+Compat 0.8.5
```
Comment (tkelman) [inline REQUIRE]: take! was 0.9.5

Comment (TotalVerb) [inline REQUIRE]: Thanks, fixed.

**Comment by TotalVerb** [comment]
Restarting the failed travis job, just in case.

**Comment by tkelman** [comment]
looked like a fluke download failure

**Comment by TotalVerb** [comment]
Looks good now.

**Pull request merged**